torch/cuda/amp/grad_scaler.py

```diff
@@ -158,7 +153,6 @@ def scale(self, outputs):
 
         # Short-circuit for the common case.
         if isinstance(outputs, torch.Tensor):
-            assert outputs.is_cuda
```

Comment:
We should not remove these asserts.  Would
```
assert output.is_cuda or output.is_xla
```
enable what you want?

Reply:
Done.